src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -193,10 +194,12 @@ export default class GeGiftEntryFormApp extends NavigationMixin(LightningElement
     prepareInMemoryDataImportForUpdate(inMemoryDataImport) {
         inMemoryDataImport.Id = this.dataImportRecord.Id;
         inMemoryDataImport[PAYMENT_METHOD__C] = this.dataImportRecord[PAYMENT_METHOD__C];
-        inMemoryDataImport[PAYMENT_STATUS__C] = this.dataImportRecord[PAYMENT_STATUS__C];
+        inMemoryDataImport[PAYMENT_STATUS__C] =
+            this._isCreditCardWidgetInDoNotChargeState ? '' : this.dataImportRecord[PAYMENT_STATUS__C];
         inMemoryDataImport[PAYMENT_DECLINED_REASON__C] =
```

Comment:
We need to clear this field as well ( I know the WI does not mention it). I forced a payment to fail by entering a negative amount, then I clicked 'Do not Charge card' and then I saved. The Declined reason ["amount: Must be greater than 0"] stayed on the DI after processing.

Reply:
I thought of the same thing too, I believe it was brought up and we were fine with leaving the decline reason. But I'll take it out.

Reply:
@vmartinov-sforg I just updated it.